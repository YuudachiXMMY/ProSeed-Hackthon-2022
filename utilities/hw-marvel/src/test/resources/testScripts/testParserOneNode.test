# Test Marvel parser can load graph with only one node

LoadGraph graph1 oneHero.tsv

ListNodes graph1
ListChildren graph1 SUPER-CSE-Student