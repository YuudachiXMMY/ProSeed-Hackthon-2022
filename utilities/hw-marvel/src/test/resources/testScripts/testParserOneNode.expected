# Test Marvel parser can load graph with only one node

loaded graph graph1

graph1 contains: SUPER-CSE-Student
the children of SUPER-CSE-Student in graph1 are: